jsx/src/components/GroupEdit/GroupEdit.jsx

```diff
@@ -3,6 +3,19 @@ import { useSelector, useDispatch } from "react-redux";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
 import GroupSelect from "../GroupSelect/GroupSelect";
+import DynamicTable from "../DynamicTable/DynamicTable";
+
+function hasDuplicates(array) {
```

Comment:
I'd simplify this and define it within the body of the GroupEdit component.

Something along the lines of
```js
const hasDuplicates = a => a.filter((e, i) => a.indexOf(e) != i).length > 0
```